main.py

```diff
@@ -33,6 +33,33 @@ def read_env_variables():
     TOTP = None if value is None else pyotp.TOTP(value)
 
 
+def purchase_steps(browser):
+    # wait until its visible and then click the purchase button
+    logger.debug('find and click on the last purchase button')
+    WebDriverWait(browser, TIMEOUT).until(
+        EC.visibility_of_element_located((By.XPATH, "//button[contains(@class,'btn-primary')]"))
+    ).click()
+
+    # wait until its visible and then click the 'I Agree" popup
+    # 'Refund and Right of Withdrawal Information' popup
+    try:
+        logger.debug('accept the conditions of refund popup')
+        WebDriverWait(browser, TIMEOUT).until(
```

Comment:
I get `TimeoutException` on this line. I don't have any refund popup, after lines 39-41 EGS shows "Thank you" message and redirects me to "Install Epic Games Launcher" page.

Reply:
Sorry, haven't noticed that it's the same code as in current master branch and it's not PR issue.
New PR code works fine for me, no errors.

Reply:
It's alright, I also added the Timeout exception to this try catch block. Hopefully it will fix your issue 👍
Keep me updated